src/server/game/Guilds/Guild.cpp

```diff
@@ -1361,25 +1361,13 @@ bool Guild::SetName(std::string const& name)
 
     ObjectGuid guid = GetGUID();
     WorldPacket data(SMSG_GUILD_RENAMED, 24 + 8 + 1);
```

Comment:
packet size ?

Reply:
Basically it's upper bound of guild name as it wasn't that big (at least there is a condition for max 24chars). I can put there name.length() if required.